algos/aho_corasick/render_test.html

```diff
@@ -0,0 +1,41 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="https://d3js.org/d3.v5.js"></script>
+    <script src="https://dagrejs.github.io/project/dagre-d3/latest/dagre-d3.min.js"></script>
+    <script src="render.js"></script>
+  <style>
```

Comment:
Is this page ever intended to be published anywhere? If so, we should move the styles to a CSS file.

Reply:
No, it's not. I've updated regenerate.py to reflect that